brands/micromax/README.md

```diff
@@ -0,0 +1,10 @@
+# Micromax
+
+* Verdict **ℹ️ "Safe for now" :trollface:**
+
+Despite their sketchy past with using their OTA updater app to [deliver adware][Micromax adware], Micromax follows the standard procedure of enabling OEM unlock in settings, then running `fastboot flashing unlock` in bootloader mode to unlock your bootloader.
```

Comment:
The boilerplate "standard unlock blah blah where you blah blah" should be replaced by a link to the generic unlock guide. `[**🔓️ Generic Unlock Guide**](/misc/generic-unlock.md)`